.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       fail-fast: false
       matrix:
         ruby: ["3.0", "3.1", "3.2", "3.3", "head"]
-        rails: ["7.0", "7.1", "main"]
+        rails: ["7.0", "current", "main"]
```

Comment:
It doesn't read that well with this `current` change compared to `7.1`. What's the goal of this change?